terraform/deployments/govuk-publishing-infrastructure/locations_api_s3.tf

```diff
@@ -2,8 +2,11 @@ resource "aws_s3_bucket" "location_api_import_csvs" {
   bucket        = "govuk-${var.govuk_environment}-locations-api-import-csvs"
   force_destroy = var.force_destroy
   tags = {
-    System = "Locations API"
-    Name   = "CSVs for importing postcode information into Locations API"
+    System      = "Locations API"
+    Name        = "CSVs for importing postcode information into Locations API"
+    Product     = "GOV.UK"
+    Environment = var.govuk_environment
```

Comment:
These should be inherited, no? Same for all the others below.
```suggestion
```

Reply:
Ah yes, Monday!

Reply:
Haha no worries :)